package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Tracks deployments of sample applications",
   "main": "app.js",
   "scripts": {
-    "postinstall": "./admin.js track && ./admin.js db put && ./admin.js ddoc put && ./admin.js clean repository_url_hash",
+    "postinstall": "bower install --allow-root && ./admin.js track && ./admin.js db put && ./admin.js ddoc put && ./admin.js clean repository_url_hash",
```

Comment:
Why `--allow-root`? I would think we should _not_ set this option. Related, we should be running Bower in non-interactive mode here (`--config.interactive=false`).